### PR TITLE
Give signature suggestions

### PR DESCRIPTION
**Pull request by andresmgot**
The signature suggestions are given when some function parameters need to be specified (it's only triggered when opening parenthesis). In this case, to maintain previous behavior, we are giving some information when the user tries to use the `$__timeFilter` macro:

Screenshot from 2022-04-07 17-50-54

### CODE REVIEW

File: src/components/Suggestions.ts [hunk 1]
```diff
@@ -105,3 +105,41 @@ export function getSuggestions(model: monacoTypes.editor.ITextModel, position: m
     suggestions: [],
   };
 }
+
+export function getSignatureHelp(
+  model: monacoTypes.editor.ITextModel,
+  position: monacoTypes.Position
+): monacoTypes.languages.ProviderResult<monacoTypes.languages.SignatureHelpResult> {
+  const word = model.getWordUntilPosition(position.delta(0, -1));
```
Comment (sarahzinger) [inline src/components/Suggestions.ts:113]: hi! mind talking through what this is doing I'm not sure I follow why we need word or what word this represents and how this is different from textUntilPosition? Couldn't we just pass 0 for the startColumn or am I misunderstanding?

**Review by sarahzinger — APPROVED** [review]
neat! I had one question for my own clarification, but I think it should be fair to ship :) maybe we should add it to changelog.

File: src/components/Suggestions.ts (hunk 1 above)
Comment (andresmgot) [reply src/components/Suggestions.ts:113]: true,  `word` may be misleading, this is actually the `wordPosition`. I just noticed that a word position actually has the property `word` (which is the word text) so I can highly simplify this :)

**Comment by andresmgot** [comment]
> maybe we should add it to changelog.

In theory, there should be "no changes" from the user perspective. I am just migrating functionality from the legacy editor to the new one (without adding anything new so far).

**Pull request merged**